include/main.h

```diff
@@ -1,64 +1,58 @@
+#ifndef HANGMAN_MAIN_H
+#define HANGMAN_MAIN_H
+
 ///////////////////////////
 // VARIABLE DECLARATIONS //
 ///////////////////////////
 
 // hangman banner
-const char *hangman = " _    _          _   _  _____ __  __          _   _   _ \n| |  | |   /\\   | \\ | |/ ____|  \\/  |   /\\   | \\ | | | |\n| |__| |  /  \\  |  \\| | |  __| \\  / |  /  \\  |  \\| | | |\n|  __  | / /\\ \\ | . ` | | |_ | |\\/| | / /\\ \\ | . ` | | |\n| |  | |/ ____ \\| |\\  | |__| | |  | |/ ____ \\| |\\  | |_|\n|_|  |_/_/    \\_\\_| \\_|\\_____|_|  |_/_/    \\_\\_| \\_| (_)\n                                                        ";
+char const* const hangman = " _    _          _   _  _____ __  __          _   _   _ \n| |  | |   /\\   | \\ | |/ ____|  \\/  |   /\\   | \\ | | | |\n| |__| |  /  \\  |  \\| | |  __| \\  / |  /  \\  |  \\| | | |\n|  __  | / /\\ \\ | . ` | | |_ | |\\/| | / /\\ \\ | . ` | | |\n| |  | |/ ____ \\| |\\  | |__| | |  | |/ ____ \\| |\\  | |_|\n|_|  |_/_/    \\_\\_| \\_|\\_____|_|  |_/_/    \\_\\_| \\_| (_)\n                                                        ";
```

Comment:
Make pointer itself `const` as well for both assurances that it won't be modified, and also to avoid multiple declaration warnings/errors in potential future multiple imports.